server/data/elite2ClientBuilder.js

```diff
@@ -0,0 +1,37 @@
+const superagent = require('superagent')
+const logger = require('../../log')
+const config = require('../config')
+
+const timeoutSpec = {
+  response: config.apis.elite2.timeout.response,
+  deadline: config.apis.elite2.timeout.deadline,
+}
+const apiUrl = config.apis.elite2.url
+module.exports = token => {
+  const userGet = userGetBuilder(token)
+  return {
+    async getOffenderDetails(bookingId) {
+      const path = `${apiUrl}api/bookings/${bookingId}?basicInfo=false`
+      return userGet({ path })
+    },
+  }
+}
+function userGetBuilder(token) {
+  return async ({ path, query = '', headers = {}, responseType = '', raw = false } = {}) => {
+    logger.info(`Get using user credentials: calling elite2api: ${path} ${query}`)
+    try {
+      const result = await superagent
+        .get(path)
```

Comment:
also be worth adding in the `keepaliveAgent` and `retries` from notm as we have a problem with requests hanging up when calling azure

Reply:
obviously not in this commit though